scipy/stats/tests/test_distributions.py

```diff
@@ -4112,3 +4112,18 @@ def test_loguniform():
     vals, _ = np.histogram(np.log10(rvs), bins=10)
     assert 900 <= vals.min() <= vals.max() <= 1100
     assert np.abs(np.median(vals) - 1000) <= 10
+
+
+def test_argus_rvs_large_chi():
+    np.random.seed(325)
+    # test that the algorithm can handle large values of chi
+    x = stats.argus.rvs(50, size=500)
+    assert_almost_equal(stats.argus(50).mean(), x.mean(), decimal=4)
+
+
+def test_argus_rvs_ratio_uniforms():
+    np.random.seed(1535)
+    # test that the ratio of uniforms algorithms works for chi > 2.611
```

Comment:
it looks like both tests test the `chi > 2.611` branch of the code, can a test for the `chi <=2.611` branch be added as well? Would be good to put these into a pytest class for argus to logically group the tests. 

You could repurpose one of these two tests to test the smaller `chi` case as it looks like they are both similar.

Reply:
chi = 1.0 is the default parameter that is tested in the basic tests (incl KS-test)

Reply:
ah, ok. Then we're already covered for `chi <= 2.611` :) 
Could we put this into a class for argus to distinguish from the surrounding tests?

Reply:
good point, i did not see that. i removed the old test:
cdf = 1 - pdf by definition in the code
that pdf is 0 outside of the bounds 0, 1 is also ensured by setting the support with a = 0, b = 1

The new tests are in a class now